### PR TITLE
[Teams] fix WARP images + links

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/connections/connect-networks/locations/setup-instructions/iOS/mdm.md [hunk 1]
```diff
@@ -2,42 +2,26 @@
 order: 2
 ---
 
-# iOS Setup Instructions (MDM)
+# iOS (MDM)
```
Comment (kkrum) [inline products/cloudflare-one/src/content/connections/connect-networks/locations/setup-instructions/iOS/mdm.md:5]: All of this should be removed and we should link them to all the MDM setup stuff in the "connect your device" section.

Comment (abracchi-tw) [reply products/cloudflare-one/src/content/connections/connect-networks/locations/setup-instructions/iOS/mdm.md:5]: Can we also remove the Manual setup counterparts?

**Review by fublok77 — APPROVED** [review]
#

**Comment by abracchi-tw** [comment]
#883 should fix this

**Pull request closed without merging**